[DOCS] Match the page_title and H1 header (#29192)
hashicorp/vault: website/content/docs/secrets/databases/index.mdx

---
layout: docs
page_title: Database secrets engine
description: >-

>    Dynamically generate database credentials based on configured roles with the database secrets engien. It works with a number of different databases through a plugin interface. 

Reviewer: ```suggestion
   Dynamically generate database credentials based on configured roles with the database secrets engine through a plugin interface to a number of different databases. 
```
